doc/modules/linear_model.rst

```diff
@@ -920,6 +927,13 @@ controls the strength of :math:`\ell_1` regularization vs. :math:`\ell_2`
 regularization. Elastic-Net is equivalent to :math:`\ell_1` when
 :math:`\rho = 1` and equivalent to :math:`\ell_2` when :math:`\rho=0`.
 
+Note that the scale of the class weights and the sample weights will influence
```

Comment:
I think this is a bit misleading. It‘s not so much the optimization process, it‘s more the optimization problem itself that is changed and therefore the solution (the coefficients) changes.
And as a user, I don’t care too much about optimization, but about penalty strength.

Reply:
Got it, thanks.